gammapy/cube/basic_cube.py

```diff
@@ -0,0 +1,73 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Functions to perform basic functions for map and cube analysis.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+from ..maps import WcsNDMap
+
+
+def fill_map_counts(event_list, ndmap):
+    """Fill a ``WcsNDMap` with events from an EventList.
+
+     The energy of the events is used for a non-spatial axis homogeneous to energy.
+     The other non-spatial axis names should have an entry in the colum names of the ``EventList``
+
+     Parameters
+     ----------
+     event_list : `~gammapy.data.EventList`
+             the input event list
+     nd_map : `~gammapy.maps.WcsNDMap`
+         Target map
+     """
```

Comment:
I think you could have this method accept a `Map` argument instead of `WcsNDMap`.  There's nothing you're doing here that specifically relies on the map being WCS-based.

Reply:
Right. I changed the docstring accordingly.
Note that the code that was committed earlier was not the right one because of a mix-up with between branches.